themes/default/layouts/case-studies/list.html

```diff
@@ -14,7 +14,7 @@ <h5 class="mx-auto max-w-xl mb-0">
             {{ $items := where (where .Site.Pages "Type" "case-studies") "Kind" "eq" "page" }}
             {{ range $index, $item := $items }}
                 <li class="m-0 p-0">
-                    <article class="{{ if lt (add $index 1) (len $items) }} pb-16 mb-16 border-b border-gray-300 {{ end }} md:flex items-start">
+                    <article class="pb-16 mb-16 border-b border-gray-300 md:flex items-start">
                         <div class="mb-8 md:mb-0 md:pt-2 px-8 md:w-1/3">
                             <img class="md:block w-40 md:w-48 mx-auto" src="{{ .Params.customer_logo }}" alt="{{ .Params.customer_name }} logo">
```

Comment:
Since you're in here, would you mind linking the images to their respective case studies as well? 🙏  It's alittle thing I overlooked when I did these originally that I haven't managed to get to yet.

Reply:
there's an issue tracking that here: #71

Reply:
👍